frontend/src/pages/analytics/segments/UserSegmentDetails.tsx

```diff
@@ -0,0 +1,659 @@
+                            if (e.jobStatus !== SegmentJobStatus.COMPLETED) {
+                              return '-';
+                            }
+
+                            const percentage =
+                              (e.segmentUserNumber / e.totalUserNumber) * 100;
+                            return percentage.toFixed(2) + '%';
+                          },
+                        },
+                        {
+                          id: 'export',
+                          header: t('analytics:segment.details.export'),
+                          cell: (e: SegmentJobStatusItem) => {
+                            return (
+                              <Button
+                                variant="icon"
+                                iconName="download"
+                                disabled={
+                                  e.jobStatus !== SegmentJobStatus.COMPLETED
+                                }
+                                onClick={async () => {
+                                  const segmentApiResponse =
+                                    await getExportFileS3Url({
+                                      projectId,
+                                      appId,
+                                      segmentId,
+                                      jobRunId: e.jobRunId,
+                                    });
+
+                                  if (segmentApiResponse.success) {
+                                    const url =
+                                      segmentApiResponse.data.presignedUrl;
+                                    const name =
+                                      defaultStr(segment?.name, 'segment') +
+                                      '.csv';
+                                    createDownloadLink(url, name);
+                                  }
+                                }}
+                              />
+                            );
+                          },
+                        },
+                      ]}
+                      items={segmentJobs}
+                    />
+                  </Container>
+                  <Box margin={{ vertical: 'm' }}>
+                    <Table
+                      loading={isSampleDataLoading}
+                      header={
+                        <Header
+                          variant="h2"
+                          description={t(
+                            'analytics:segment.details.segmentUserSampleDesc'
+                          )}
+                        >
+                          {t('analytics:segment.details.segmentUserSample')}
+                        </Header>
+                      }
+                      variant="container"
+                      columnDefinitions={[
+                        {
+                          id: 'user_pseudo_id',
+                          header: 'User pseudo id',
+                          cell: (e: SegmentUserItem) => e.user_pseudo_id,
```

Comment:
i18n? for this and below headers

Reply:
Confirmed with Robin, we don't need i18n for these headers since they are mapped to the exported segment csv file headers.